joomla/joomla-cms
# [#26922] fix - Repeatable-table field - UX issue

Pull Request for Issue # .

### Summary of Changes
Added "table-responsive" class for repeatable table field


### Testing Instructions
Go to the plugin: Fields radio
Add multiple options to a radio field.
Check add, remove button UX

### Expected result



### Actual result



### Documentation Changes Required
